### ISSUE TITLE
Adding a new feature

### ISSUE THREAD

**Issue by gyani-codes** [LorenzoBe/LeetPlug#9]
Hi there I am thinking of adding a new column in the field to store the tag of the problem too in the csv, so that later we can analyze our data in to track the progress.

**Comment by LorenzoBe** [comment]
Hi @vishal-codes, sorry but I didn't understand very well. What should be the source of the tag? The LeetCode page? Are there any tag connected to the problem?
Or probably you would like to add it directly in the LeetPlug page? Unfortunately this would require quite a lot of work, currently the the web page is just a visualiser without any posibility to write new data/metadata/tag

**Comment by gyani-codes** [comment]
Hi @LorenzoBe the tag of the problem would pe the category of the problem like stack , queue , linked list , greedy . So that user can see a piechart of how many problems he has solved in the particular category.

**Comment by LorenzoBe** [comment]
Now I can see! So you mean to submit also the keywords in "Related Topics" field of the problem. Thanks, this would be nice.
It would require some change to both the browser extension and the web service. Let's see if some other would like the feature. If you have time, I also would accept a Pull request :)

**Comment by gyani-codes** [comment]
@LorenzoBe  Yes Sure , I have been currently busy with my university exams I will contact you after 16th Feb. Thanks
